src/utils/app_config.py

```diff
@@ -0,0 +1,7 @@
+import os
+
+def database_url():
+    database_url = os.getenv("TAGGING-SUGGESTER-DATABASE-URL")
+    if database_url is None:
+        database_url = "postgres://taggingsuggester@localhost:5432/tagging_suggester_development"
```

Comment:
Again, do we need another database specified here for production?

Reply:
I don't think so - AFAIK sqlalchemy doesn't have a concept of environments

Reply:
As discussed